src/components/actions/processTrigger.ts

```diff
@@ -111,6 +111,12 @@ export async function processTrigger(payload: TriggerPayload): Promise<ActionRes
         actionFailed = action.action_code
       }
     } catch (err) {
+      actionOutputs.push({
+        action: action.action_code,
+        status: ActionQueueStatus.Fail,
+        output: null,
+        errorLog: err.message,
+      })
```

Comment:
This was also missing, and is basically the opposite of what I fixed the other day. It was only returning information for failed actions if they returned a "FAIL" status, not for ones that throw an error.

Now it behaves the same for both.